Profiler/Profiler.example.yml

```diff
@@ -7,7 +7,7 @@ match:
       enabled: false
       # Trace files will be stored and archived here
       targetdir: C:\Users\Public\Traces
-      # Light mode is recommended.
+      # Light mode is recommended and enabled by default.
```

Comment:
Please add a comment here that light mode needs to be disabled in case the Native Image Cache is used

Reply:
The line below this change already reflects this:  `Turn it off if you use ngen.exe for your assemblies`

Reply:
ah sorry, I hadn't seen that. my bad